index.js

```diff
@@ -30,4 +163,6 @@ burger.addEventListener('click', () => {
 
 navigationMenu.addEventListener('click', () => {
      navigation.classList.remove('active');
-})
\ No newline at end of file
+})
+
+
```

Comment:
Co do komentarzy, to moim zdaniem to jest zla praktyka, zaczniesz cos zmieniac w kodzie, nie aktualizujac tych komentarzy i bedziesz miala wiekszy balagan niz bedziesz w stanie ogarnac, sprobuj je wywalic, i napisac tak kod, z uzyciem malych drobnych funkcji, generalnie komentarzom mówimy NIE